dev/workflow/applications/gcafs_cycled.py

```diff
@@ -0,0 +1,241 @@
+        """
+        run_options = super()._get_run_options(conf)
+
+        for run in self.runs:
+            base = conf.parse_config('config.base', RUN=run)
+
+            run_options[run]['do_hybvar'] = base.get('DOHYBVAR', False)
+            run_options[run]['nens'] = base.get('NMEM_ENS', 0)
+            if run_options[run]['do_hybvar']:
+                run_options[run]['lobsdiag_forenkf'] = base.get('lobsdiag_forenkf', False)
+
+            run_options[run]['do_jediatmvar'] = base.get('DO_JEDIATMVAR', False)
+            run_options[run]['do_jediatmens'] = base.get('DO_JEDIATMENS', False)
+            run_options[run]['do_mergensst'] = base.get('DO_MERGENSST', False)
+
+        return run_options
+
+    def _get_app_configs(self, run):
+        """
+        Returns the config files that are involved in the GCAFS cycled app.
+
+        Parameters
+        ----------
+        run : str
+            Name of the run configuration to process
+
+        Returns
+        -------
+        list
+            List of configuration file names needed for the specified run
+        """
+        options = self.run_options[run]
+
+        configs = ['fetch']
+
+        configs += ['offlineanl']
+
+        # Add GCAFS-specific aerosol configs by default
+        if options['do_aero_fcst']:
+            configs += ['aero', 'prep_emissions']
+            # Don't include aerosol_init for cycled runs
+            # aerosol_init is only needed for forecast-only mode
+
+        configs += ['stage_ic', 'sfcanl', 'fcst', 'upp', 'atmos_products', 'arch_vrfy', 'cleanup']
+
+        if options['do_archcom']:
+            configs += ['arch_tars']
+
+        if options['do_hybvar']:
+            print("WARNING: Hyb-Var not yet supported for GCAFS")
+        # if options['do_hybvar']:
+        #     if options['do_jediatmens']:
+        #         configs += ['atmensanlinit', 'atmensanlobs', 'atmensanlsol',
+        #                     'atmensanlletkf', 'atmensanlfv3inc', 'atmensanlfinal',
+        #                     'ecen_fv3jedi']
+
+        #     configs += ['esfc', 'efcs', 'epos', 'earc_vrfy']
+
+        #     if options['do_archcom']:
+        #         configs += ['earc_tars']
+
+        if options['do_metp']:
+            configs += ['metp']
+
+        if options['do_aero_anl']:
+            configs += ['aeroanlgenb', 'aeroanlinit', 'aeroanlvar', 'aeroanlfinal']
+            configs += ['prepobsaero']
+
+        if options['do_anlstat']:
+            configs += ['anlstat']
+
+        if options['do_globusarch']:
+            configs += ['globus']
+            # TODO Enable when the globus archiving feature is available for ensembles
+            if options['do_hybvar']:
+                print("WARNING Globus archiving is currently only possible for deterministic members")
+                print("        Ensemble members will NOT be archived with this option!!")
+            # if options['do_hybvar']:
+            #     configs += ['globus_earc']
```

Comment:
The `globus_earc` capability is now available, though `hyb-var` is not for GCAFS.

```suggestion
            # TODO Enable when GCAFS hybvar is enabled.
            #if options['do_hybvar']:
            #    print("WARNING Globus archiving is currently only possible for deterministic members")
            #   print("        Ensemble members will NOT be archived with this option!!")
            # if options['do_hybvar']:
            #     configs += ['globus_earc']
```